bluemira/base/constants.py

```diff
@@ -223,6 +232,13 @@ def _transform(
     ureg.Quantity("m_e").to("g") * ureg.Quantity("avogadro_constant").to_base_units()
 ).magnitude
 
+
+# electron mass [kg]
+ELECTRON_MASS = ureg.Quantity("m_e").to("g").to_base_units().magnitude
+
+# proton mass [kg]
+PROTON_MASS = ureg.Quantity("m_p").to("g").to_base_units().magnitude
```

Comment:
On a similar vein to @hsaunders1904 just passing through. You could directly do `.to('kg')` or just `to_base_units()`.

```suggestion
# electron mass [kg]
ELECTRON_MASS = ureg.Quantity("m_e").to_base_units().magnitude

# proton mass [kg]
PROTON_MASS = ureg.Quantity("m_p").to_base_units().magnitude
```